router/router.go

```diff
@@ -318,6 +317,29 @@ func (router *Router) handleInvokeEvent(space string, functionID function.ID, pa
 	}
 }
 
+func determineErrorMessage(err error) string {
```

Comment:
This function should not be defined in router package. It should be defined in function package. This is where providers specific logic should live.